Erro ao receber arquivo com Justificativa superior a 1000 caracteres

**Descreva o erro**
O módulo está chamando o serviço de recusa e informando um texto maior do que 1000 caracteres

-----
**Log de Erros**

```WARN  2022-08-23 10:31:43,148 [[pen-interoperabilidade-api].connector.http.mule.default.receiver.04] org.apache.cxf.phase.PhaseInterceptorChain: Interceptor for {http://pen.planejamento.gov.br/interoperabilidade/soap/v2}InteroperabilidadePENService#{http://pen.planejamento.gov.br/interoperabilidade/soap/v2}recusarTramite has thrown exception, unwinding now
org.apache.cxf.interceptor.Fault: Unmarshalling Error: cvc-maxLength-valid: Value 'Descrição:
SoapFault exception: [Client] looks like we got no XML document in /opt/sei/web/modulos/pen/rn/ProcessoEletronicoRN.php:1100
Stack trace:
#0 /opt/sei/web/modulos/pen/rn/ProcessoEletronicoRN.php(1100): SoapClient->__call(\'receberComponen...\', Array)
#1 /opt/sei/web/modulos/pen/rn/ProcessoEletronicoRN.php(1100): BeSimple\SoapClient\SoapClient->receberComponenteDigital(Object(stdClass))
#2 /opt/sei/web/modulos/pen/rn/ProcessoEletronicoRN.php(1647): ProcessoEletronicoRN->{closure}(Object(BeSimple\SoapClient\SoapClient))
#3 /opt/sei/web/modulos/pen/rn/ProcessoEletronicoRN.php(1101): ProcessoEletronicoRN->tentarNovamenteSobErroHTTP(Object(Closure))
#4 /opt/sei/web/modulos/pen/rn/ReceberProcedimentoRN.php(2250): ProcessoEletronicoRN->receberComponenteDigital(632958, \'IpG40wg2G9JGI1w...\', \'23538.013761/20...\', Object(stdClass))
#5 /opt/sei/web/modulos/pen/rn/ReceberProcedimentoRN.php(269): ReceberProcedimentoRN->receberComponenenteDigitalParticionado(\'IpG40wg2G9JGI1w...\', 5242880, 789659, \'5\', 2, 632958, Object(stdClass), Array)
#6 /opt/sei/web/modulos/pen/rn/ReceberProcedimentoRN.php(82): ReceberProcedimentoRN->baixarComponentesDigitais(Object(stdClass), Object(stdClass))
#7 /opt/sei/web/modulos/pen/rn/ProcessarPendenciasRN.php(180): ReceberProcedimentoRN->receberProcedimento(632958)
#8 /opt/sei/web/modulos/pen/rn/PendenciasTramiteRN.php(368): ProcessarPendenciasRN->receberProcedimento(\'632958\')
#9 /opt/sei/web/modulos/pen/rn/PendenciasTramiteRN.php(342): PendenciasTramiteRN->enviarPendenciaProcessamentoDireto(Object(PendenciaDTO))
#10 /opt/sei/web/modulos/pen/rn/PendenciasTramiteRN.php(101): PendenciasTramiteRN->enviarPendenciaProcessamento(Object(PendenciaDTO), false)
#11 /opt/sei/scripts/mod-pen/MonitoramentoTarefasPEN.php(30): PendenciasTramiteRN->encaminharPendencias(false, false, false)
#12 {main}

Mensagem:
looks like we got no XML document

Trilha de Processamento:
#0 /opt/sei/web/modulos/pen/rn/ProcessoEletronicoRN.php(1100): SoapClient->__call(\'receberComponen...\', Array)
#1 /opt/sei/web/modulos/pen/rn/ProcessoEletronicoRN.php(1100): BeSimple\SoapClient\SoapClient->receberComponenteDigital(Object(stdClass))
#2 /opt/sei/web/modulos/pen/rn/ProcessoEletronicoRN.php(1647): ProcessoEletronicoRN->{closure}(Object(BeSimple\SoapClient\SoapClient))
#3 /opt/sei/web/modulos/pen/rn/ProcessoEletronicoRN.php(1101): ProcessoEletronicoRN->tentarNovamenteSobErroHTTP(Object(Closure))
#4 /opt/sei/web/modulos/pen/rn/ReceberProcedimentoRN.php(2250): ProcessoEletronicoRN->receberComponenteDigital(632958, \'IpG40wg2G9JGI1w...\', \'23538.013761/20...\', Object(stdClass))
#5 /opt/sei/web/modulos/pen/rn/ReceberProcedimentoRN.php(269): ReceberProcedimentoRN->receberComponenenteDigitalParticionado(\'IpG40wg2G9JGI1w...\', 5242880, 789659, \'5\', 2, 632958, Object(stdClass), Array)
#6 /opt/sei/web/modulos/pen/rn/ReceberProcedimentoRN.php(82): ReceberProcedimentoRN->baixarComponentesDigitais(Object(stdClass), Object(stdClass))
#7 /opt/sei/web/modulos/pen/rn/ProcessarPendenciasRN.php(180): ReceberProcedimentoRN->receberProcedimento(632958)
#8 /opt/sei/web/modulos/pen/rn/PendenciasTramiteRN.php(368): ProcessarPendenciasRN->receberProcedimento(\'632958\')
#9 /opt/sei/web/modulos/pen/rn/PendenciasTramiteRN.php(342): PendenciasTramiteRN->enviarPendenciaProcessamentoDireto(Object(PendenciaDTO))
#10 /opt/sei/web/modulos/pen/rn/PendenciasTramiteRN.php(101): PendenciasTramiteRN->enviarPendenciaProcessamento(Object(PendenciaDTO), false)
#11 /opt/sei/scripts/mod-pen/MonitoramentoTarefasPEN.php(30): PendenciasTramiteRN->encaminharPendencias(false, false, false)
#12 {main}

' with length = '3625' is not facet-valid with respect to maxLength '1000' for type 'justificativaDaRecusaDeTramite'. 
	at org.apache.cxf.jaxb.JAXBEncoderDecoder.unmarshall(JAXBEncoderDecoder.java:812) ~[cxf-rt-databinding-jaxb-2.5.9.jar:2.5.9]
	at org.apache.cxf.jaxb.JAXBEncoderDecoder.unmarshall(JAXBEncoderDecoder.java:633) ~[cxf-rt-databinding-jaxb-2.5.9.jar:2.5.9]
	at org.apache.cxf.jaxb.io.DataReaderImpl.read(DataReaderImpl.java:157) ~[cxf-rt-databinding-jaxb-2.5.9.jar:2.5.9]
	at org.apache.cxf.interceptor.DocLiteralInInterceptor.handleMessage(DocLiteralInInterceptor.java:110) ~[cxf-rt-core-2.5.9.jar:2.5.9]
	at org.apache.cxf.phase.PhaseInterceptorChain.doIntercept(PhaseInterceptorChain.java:263) ~[cxf-api-2.5.9.jar:2.5.9]
	at org.apache.cxf.transport.ChainInitiationObserver.onMessage(ChainInitiationObserver.java:122) ~[cxf-rt-core-2.5.9.jar:2.5.9]
	at org.mule.module.cxf.CxfInboundMessageProcessor.sendToDestination(CxfInboundMessageProcessor.java:338) ~[mule-module-cxf-3.6.1.jar:3.6.1]
	at org.mule.module.cxf.CxfInboundMessageProcessor.process(CxfInboundMessageProcessor.java:144) ~[mule-module-cxf-3.6.1.jar:3.6.1]
	at org.mule.module.cxf.config.FlowConfiguringMessageProcessor.process(FlowConfiguringMessageProcessor.java:48) ~[mule-module-cxf-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.DefaultMessageProcessorChain.doProcess(DefaultMessageProcessorChain.java:94) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.AbstractMessageProcessorChain.process(AbstractMessageProcessorChain.java:69) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.InterceptingChainLifecycleWrapper.doProcess(InterceptingChainLifecycleWrapper.java:50) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.AbstractMessageProcessorChain.process(AbstractMessageProcessorChain.java:69) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.InterceptingChainLifecycleWrapper.access$001(InterceptingChainLifecycleWrapper.java:22) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.InterceptingChainLifecycleWrapper$1.process(InterceptingChainLifecycleWrapper.java:66) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.InterceptingChainLifecycleWrapper.process(InterceptingChainLifecycleWrapper.java:61) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.DefaultMessageProcessorChain.doProcess(DefaultMessageProcessorChain.java:94) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.AbstractMessageProcessorChain.process(AbstractMessageProcessorChain.java:69) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.AbstractInterceptingMessageProcessorBase.processNext(AbstractInterceptingMessageProcessorBase.java:102) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.AsyncInterceptingMessageProcessor.process(AsyncInterceptingMessageProcessor.java:102) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.AbstractInterceptingMessageProcessorBase.processNext(AbstractInterceptingMessageProcessorBase.java:102) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.construct.DynamicPipelineMessageProcessor.process(DynamicPipelineMessageProcessor.java:54) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.DefaultMessageProcessorChain.doProcess(DefaultMessageProcessorChain.java:94) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.AbstractMessageProcessorChain.process(AbstractMessageProcessorChain.java:69) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.AbstractInterceptingMessageProcessorBase.processNext(AbstractInterceptingMessageProcessorBase.java:102) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.interceptor.AbstractEnvelopeInterceptor.process(AbstractEnvelopeInterceptor.java:51) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.AbstractInterceptingMessageProcessorBase.processNext(AbstractInterceptingMessageProcessorBase.java:102) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.AbstractFilteringMessageProcessor.process(AbstractFilteringMessageProcessor.java:40) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.AbstractInterceptingMessageProcessorBase.processNext(AbstractInterceptingMessageProcessorBase.java:102) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.construct.AbstractPipeline$1.process(AbstractPipeline.java:109) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.DefaultMessageProcessorChain.doProcess(DefaultMessageProcessorChain.java:94) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.AbstractMessageProcessorChain.process(AbstractMessageProcessorChain.java:69) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.InterceptingChainLifecycleWrapper.doProcess(InterceptingChainLifecycleWrapper.java:50) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.AbstractMessageProcessorChain.process(AbstractMessageProcessorChain.java:69) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.InterceptingChainLifecycleWrapper.access$001(InterceptingChainLifecycleWrapper.java:22) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.InterceptingChainLifecycleWrapper$1.process(InterceptingChainLifecycleWrapper.java:66) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.InterceptingChainLifecycleWrapper.process(InterceptingChainLifecycleWrapper.java:61) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.construct.AbstractPipeline$3.process(AbstractPipeline.java:207) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.SimpleMessageProcessorChain.doProcess(SimpleMessageProcessorChain.java:43) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.AbstractMessageProcessorChain.process(AbstractMessageProcessorChain.java:69) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.InterceptingChainLifecycleWrapper.doProcess(InterceptingChainLifecycleWrapper.java:50) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.AbstractMessageProcessorChain.process(AbstractMessageProcessorChain.java:69) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.InterceptingChainLifecycleWrapper.access$001(InterceptingChainLifecycleWrapper.java:22) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.InterceptingChainLifecycleWrapper$1.process(InterceptingChainLifecycleWrapper.java:66) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.InterceptingChainLifecycleWrapper.process(InterceptingChainLifecycleWrapper.java:61) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.SimpleMessageProcessorChain.doProcess(SimpleMessageProcessorChain.java:43) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.AbstractMessageProcessorChain.process(AbstractMessageProcessorChain.java:69) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.InterceptingChainLifecycleWrapper.doProcess(InterceptingChainLifecycleWrapper.java:50) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.AbstractMessageProcessorChain.process(AbstractMessageProcessorChain.java:69) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.InterceptingChainLifecycleWrapper.access$001(InterceptingChainLifecycleWrapper.java:22) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.InterceptingChainLifecycleWrapper$1.process(InterceptingChainLifecycleWrapper.java:66) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.chain.InterceptingChainLifecycleWrapper.process(InterceptingChainLifecycleWrapper.java:61) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.transport.AbstractMessageReceiver.routeEvent(AbstractMessageReceiver.java:511) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.transport.AbstractTransportMessageProcessTemplate.routeEvent(AbstractTransportMessageProcessTemplate.java:72) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.FlowProcessingPhase$1$1.process(FlowProcessingPhase.java:76) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.FlowProcessingPhase$1$1.process(FlowProcessingPhase.java:63) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExecuteCallbackInterceptor.execute(ExecuteCallbackInterceptor.java:16) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.HandleExceptionInterceptor.execute(HandleExceptionInterceptor.java:30) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.HandleExceptionInterceptor.execute(HandleExceptionInterceptor.java:14) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.BeginAndResolveTransactionInterceptor.execute(BeginAndResolveTransactionInterceptor.java:54) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ResolvePreviousTransactionInterceptor.execute(ResolvePreviousTransactionInterceptor.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.SuspendXaTransactionInterceptor.execute(SuspendXaTransactionInterceptor.java:50) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ValidateTransactionalStateInterceptor.execute(ValidateTransactionalStateInterceptor.java:40) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.IsolateCurrentTransactionInterceptor.execute(IsolateCurrentTransactionInterceptor.java:41) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExternalTransactionInterceptor.execute(ExternalTransactionInterceptor.java:48) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.RethrowExceptionInterceptor.execute(RethrowExceptionInterceptor.java:28) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.RethrowExceptionInterceptor.execute(RethrowExceptionInterceptor.java:13) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.TransactionalErrorHandlingExecutionTemplate.execute(TransactionalErrorHandlingExecutionTemplate.java:109) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.FlowProcessingPhase$1.run(FlowProcessingPhase.java:62) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.transport.TrackingWorkManager$TrackeableWork.run(TrackingWorkManager.java:267) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.work.WorkerContext.run(WorkerContext.java:286) ~[mule-core-3.6.1.jar:3.6.1]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1152) [?:1.7.0_201]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:622) [?:1.7.0_201]
	at java.lang.Thread.run(Thread.java:748) [?:1.7.0_201]
Caused by: javax.xml.bind.UnmarshalException
	at com.sun.xml.bind.v2.runtime.unmarshaller.UnmarshallerImpl.handleStreamException(UnmarshallerImpl.java:425) ~[jaxb-impl-2.1.9.jar:2.1.9]
	at com.sun.xml.bind.v2.runtime.unmarshaller.UnmarshallerImpl.unmarshal0(UnmarshallerImpl.java:362) ~[jaxb-impl-2.1.9.jar:2.1.9]
	at com.sun.xml.bind.v2.runtime.unmarshaller.UnmarshallerImpl.unmarshal(UnmarshallerImpl.java:339) ~[jaxb-impl-2.1.9.jar:2.1.9]
	at org.apache.cxf.jaxb.JAXBEncoderDecoder.doUnmarshal(JAXBEncoderDecoder.java:773) ~[cxf-rt-databinding-jaxb-2.5.9.jar:2.5.9]
	at org.apache.cxf.jaxb.JAXBEncoderDecoder.access$100(JAXBEncoderDecoder.java:93) ~[cxf-rt-databinding-jaxb-2.5.9.jar:2.5.9]
	at org.apache.cxf.jaxb.JAXBEncoderDecoder$1.run(JAXBEncoderDecoder.java:801) ~[cxf-rt-databinding-jaxb-2.5.9.jar:2.5.9]
	at java.security.AccessController.doPrivileged(Native Method) ~[?:1.7.0_201]
	at org.apache.cxf.jaxb.JAXBEncoderDecoder.unmarshall(JAXBEncoderDecoder.java:799) ~[cxf-rt-databinding-jaxb-2.5.9.jar:2.5.9]
	at org.apache.cxf.jaxb.JAXBEncoderDecoder.unmarshall(JAXBEncoderDecoder.java:633) ~[cxf-rt-databinding-jaxb-2.5.9.jar:2.5.9]
	at org.apache.cxf.jaxb.io.DataReaderImpl.read(DataReaderImpl.java:157) ~[cxf-rt-databinding-jaxb-2.5.9.jar:2.5.9]
	at org.apache.cxf.interceptor.DocLiteralInInterceptor.handleMessage(DocLiteralInInterceptor.java:110) ~[cxf-rt-core-2.5.9.jar:2.5.9]
	at org.apache.cxf.phase.PhaseInterceptorChain.doIntercept(PhaseInterceptorChain.java:263) ~[cxf-api-2.5.9.jar:2.5.9]
	at org.apache.cxf.transport.ChainInitiationObserver.onMessage(ChainInitiationObserver.java:122) ~[cxf-rt-core-2.5.9.jar:2.5.9]
	at org.mule.module.cxf.CxfInboundMessageProcessor.sendToDestination(CxfInboundMessageProcessor.java:338) ~[mule-module-cxf-3.6.1.jar:3.6.1]
	at org.mule.module.cxf.CxfInboundMessageProcessor.process(CxfInboundMessageProcessor.java:144) ~[mule-module-cxf-3.6.1.jar:3.6.1]
	at org.mule.module.cxf.config.FlowConfiguringMessageProcessor.process(FlowConfiguringMessageProcessor.java:48) ~[mule-module-cxf-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.AsyncInterceptingMessageProcessor.process(AsyncInterceptingMessageProcessor.java:102) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.construct.DynamicPipelineMessageProcessor.process(DynamicPipelineMessageProcessor.java:54) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.interceptor.AbstractEnvelopeInterceptor.process(AbstractEnvelopeInterceptor.java:51) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.AbstractFilteringMessageProcessor.process(AbstractFilteringMessageProcessor.java:40) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.construct.AbstractPipeline$1.process(AbstractPipeline.java:109) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.construct.AbstractPipeline$3.process(AbstractPipeline.java:207) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	... 22 more
Caused by: org.xml.sax.SAXParseException: cvc-maxLength-valid: Value 'Descrição:
SoapFault exception: [Client] looks like we got no XML document in /opt/sei/web/modulos/pen/rn/ProcessoEletronicoRN.php:1100
Stack trace:
#0 /opt/sei/web/modulos/pen/rn/ProcessoEletronicoRN.php(1100): SoapClient->__call(\'receberComponen...\', Array)
#1 /opt/sei/web/modulos/pen/rn/ProcessoEletronicoRN.php(1100): BeSimple\SoapClient\SoapClient->receberComponenteDigital(Object(stdClass))
#2 /opt/sei/web/modulos/pen/rn/ProcessoEletronicoRN.php(1647): ProcessoEletronicoRN->{closure}(Object(BeSimple\SoapClient\SoapClient))
#3 /opt/sei/web/modulos/pen/rn/ProcessoEletronicoRN.php(1101): ProcessoEletronicoRN->tentarNovamenteSobErroHTTP(Object(Closure))
#4 /opt/sei/web/modulos/pen/rn/ReceberProcedimentoRN.php(2250): ProcessoEletronicoRN->receberComponenteDigital(632958, \'IpG40wg2G9JGI1w...\', \'23538.013761/20...\', Object(stdClass))
#5 /opt/sei/web/modulos/pen/rn/ReceberProcedimentoRN.php(269): ReceberProcedimentoRN->receberComponenenteDigitalParticionado(\'IpG40wg2G9JGI1w...\', 5242880, 789659, \'5\', 2, 632958, Object(stdClass), Array)
#6 /opt/sei/web/modulos/pen/rn/ReceberProcedimentoRN.php(82): ReceberProcedimentoRN->baixarComponentesDigitais(Object(stdClass), Object(stdClass))
#7 /opt/sei/web/modulos/pen/rn/ProcessarPendenciasRN.php(180): ReceberProcedimentoRN->receberProcedimento(632958)
#8 /opt/sei/web/modulos/pen/rn/PendenciasTramiteRN.php(368): ProcessarPendenciasRN->receberProcedimento(\'632958\')
#9 /opt/sei/web/modulos/pen/rn/PendenciasTramiteRN.php(342): PendenciasTramiteRN->enviarPendenciaProcessamentoDireto(Object(PendenciaDTO))
#10 /opt/sei/web/modulos/pen/rn/PendenciasTramiteRN.php(101): PendenciasTramiteRN->enviarPendenciaProcessamento(Object(PendenciaDTO), false)
#11 /opt/sei/scripts/mod-pen/MonitoramentoTarefasPEN.php(30): PendenciasTramiteRN->encaminharPendencias(false, false, false)
#12 {main}

Mensagem:
looks like we got no XML document

Trilha de Processamento:
#0 /opt/sei/web/modulos/pen/rn/ProcessoEletronicoRN.php(1100): SoapClient->__call(\'receberComponen...\', Array)
#1 /opt/sei/web/modulos/pen/rn/ProcessoEletronicoRN.php(1100): BeSimple\SoapClient\SoapClient->receberComponenteDigital(Object(stdClass))
#2 /opt/sei/web/modulos/pen/rn/ProcessoEletronicoRN.php(1647): ProcessoEletronicoRN->{closure}(Object(BeSimple\SoapClient\SoapClient))
#3 /opt/sei/web/modulos/pen/rn/ProcessoEletronicoRN.php(1101): ProcessoEletronicoRN->tentarNovamenteSobErroHTTP(Object(Closure))
#4 /opt/sei/web/modulos/pen/rn/ReceberProcedimentoRN.php(2250): ProcessoEletronicoRN->receberComponenteDigital(632958, \'IpG40wg2G9JGI1w...\', \'23538.013761/20...\', Object(stdClass))
#5 /opt/sei/web/modulos/pen/rn/ReceberProcedimentoRN.php(269): ReceberProcedimentoRN->receberComponenenteDigitalParticionado(\'IpG40wg2G9JGI1w...\', 5242880, 789659, \'5\', 2, 632958, Object(stdClass), Array)
#6 /opt/sei/web/modulos/pen/rn/ReceberProcedimentoRN.php(82): ReceberProcedimentoRN->baixarComponentesDigitais(Object(stdClass), Object(stdClass))
#7 /opt/sei/web/modulos/pen/rn/ProcessarPendenciasRN.php(180): ReceberProcedimentoRN->receberProcedimento(632958)
#8 /opt/sei/web/modulos/pen/rn/PendenciasTramiteRN.php(368): ProcessarPendenciasRN->receberProcedimento(\'632958\')
#9 /opt/sei/web/modulos/pen/rn/PendenciasTramiteRN.php(342): PendenciasTramiteRN->enviarPendenciaProcessamentoDireto(Object(PendenciaDTO))
#10 /opt/sei/web/modulos/pen/rn/PendenciasTramiteRN.php(101): PendenciasTramiteRN->enviarPendenciaProcessamento(Object(PendenciaDTO), false)
#11 /opt/sei/scripts/mod-pen/MonitoramentoTarefasPEN.php(30): PendenciasTramiteRN->encaminharPendencias(false, false, false)
#12 {main}

' with length = '3625' is not facet-valid with respect to maxLength '1000' for type 'justificativaDaRecusaDeTramite'.
	at org.apache.xerces.util.ErrorHandlerWrapper.createSAXParseException(Unknown Source) ~[?:?]
	at org.apache.xerces.util.ErrorHandlerWrapper.error(Unknown Source) ~[?:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[?:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[?:?]
	at org.apache.xerces.impl.XMLErrorReporter.reportError(Unknown Source) ~[?:?]
	at org.apache.xerces.impl.xs.XMLSchemaValidator$XSIErrorReporter.reportError(Unknown Source) ~[?:?]
	at org.apache.xerces.impl.xs.XMLSchemaValidator.reportSchemaError(Unknown Source) ~[?:?]
	at org.apache.xerces.impl.xs.XMLSchemaValidator.elementLocallyValidType(Unknown Source) ~[?:?]
	at org.apache.xerces.impl.xs.XMLSchemaValidator.processElementContent(Unknown Source) ~[?:?]
	at org.apache.xerces.impl.xs.XMLSchemaValidator.handleEndElement(Unknown Source) ~[?:?]
	at org.apache.xerces.impl.xs.XMLSchemaValidator.endElement(Unknown Source) ~[?:?]
	at org.apache.xerces.jaxp.validation.ValidatorHandlerImpl.endElement(Unknown Source) ~[?:?]
	at com.sun.xml.bind.v2.runtime.unmarshaller.ValidatingUnmarshaller.endElement(ValidatingUnmarshaller.java:94) ~[jaxb-impl-2.1.9.jar:2.1.9]
	at com.sun.xml.bind.v2.runtime.unmarshaller.StAXStreamConnector.handleEndElement(StAXStreamConnector.java:250) ~[jaxb-impl-2.1.9.jar:2.1.9]
	at com.sun.xml.bind.v2.runtime.unmarshaller.StAXStreamConnector.bridge(StAXStreamConnector.java:214) ~[jaxb-impl-2.1.9.jar:2.1.9]
	at com.sun.xml.bind.v2.runtime.unmarshaller.UnmarshallerImpl.unmarshal0(UnmarshallerImpl.java:360) ~[jaxb-impl-2.1.9.jar:2.1.9]
	at com.sun.xml.bind.v2.runtime.unmarshaller.UnmarshallerImpl.unmarshal(UnmarshallerImpl.java:339) ~[jaxb-impl-2.1.9.jar:2.1.9]
	at org.apache.cxf.jaxb.JAXBEncoderDecoder.doUnmarshal(JAXBEncoderDecoder.java:773) ~[cxf-rt-databinding-jaxb-2.5.9.jar:2.5.9]
	at org.apache.cxf.jaxb.JAXBEncoderDecoder.access$100(JAXBEncoderDecoder.java:93) ~[cxf-rt-databinding-jaxb-2.5.9.jar:2.5.9]
	at org.apache.cxf.jaxb.JAXBEncoderDecoder$1.run(JAXBEncoderDecoder.java:801) ~[cxf-rt-databinding-jaxb-2.5.9.jar:2.5.9]
	at java.security.AccessController.doPrivileged(Native Method) ~[?:1.7.0_201]
	at org.apache.cxf.jaxb.JAXBEncoderDecoder.unmarshall(JAXBEncoderDecoder.java:799) ~[cxf-rt-databinding-jaxb-2.5.9.jar:2.5.9]
	at org.apache.cxf.jaxb.JAXBEncoderDecoder.unmarshall(JAXBEncoderDecoder.java:633) ~[cxf-rt-databinding-jaxb-2.5.9.jar:2.5.9]
	at org.apache.cxf.jaxb.io.DataReaderImpl.read(DataReaderImpl.java:157) ~[cxf-rt-databinding-jaxb-2.5.9.jar:2.5.9]
	at org.apache.cxf.interceptor.DocLiteralInInterceptor.handleMessage(DocLiteralInInterceptor.java:110) ~[cxf-rt-core-2.5.9.jar:2.5.9]
	at org.apache.cxf.phase.PhaseInterceptorChain.doIntercept(PhaseInterceptorChain.java:263) ~[cxf-api-2.5.9.jar:2.5.9]
	at org.apache.cxf.transport.ChainInitiationObserver.onMessage(ChainInitiationObserver.java:122) ~[cxf-rt-core-2.5.9.jar:2.5.9]
	at org.mule.module.cxf.CxfInboundMessageProcessor.sendToDestination(CxfInboundMessageProcessor.java:338) ~[mule-module-cxf-3.6.1.jar:3.6.1]
	at org.mule.module.cxf.CxfInboundMessageProcessor.process(CxfInboundMessageProcessor.java:144) ~[mule-module-cxf-3.6.1.jar:3.6.1]
	at org.mule.module.cxf.config.FlowConfiguringMessageProcessor.process(FlowConfiguringMessageProcessor.java:48) ~[mule-module-cxf-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.AsyncInterceptingMessageProcessor.process(AsyncInterceptingMessageProcessor.java:102) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.construct.DynamicPipelineMessageProcessor.process(DynamicPipelineMessageProcessor.java:54) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.interceptor.AbstractEnvelopeInterceptor.process(AbstractEnvelopeInterceptor.java:51) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.processor.AbstractFilteringMessageProcessor.process(AbstractFilteringMessageProcessor.java:40) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.construct.AbstractPipeline$1.process(AbstractPipeline.java:109) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.construct.AbstractPipeline$3.process(AbstractPipeline.java:207) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.ExceptionToMessagingExceptionExecutionInterceptor.execute(ExceptionToMessagingExceptionExecutionInterceptor.java:24) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorNotificationExecutionInterceptor.execute(MessageProcessorNotificationExecutionInterceptor.java:58) ~[mule-core-3.6.1.jar:3.6.1]
	at org.mule.execution.MessageProcessorExecutionTemplate.execute(MessageProcessorExecutionTemplate.java:44) ~[mule-core-3.6.1.jar:3.6.1]
	... 22 more´´´

--- a/src/rn/ProcessoEletronicoRN.php
+++ b/src/rn/ProcessoEletronicoRN.php
@@ -1626,7 +1626,7 @@ class ProcessoEletronicoRN extends InfraRN
             $parametros = new stdClass();
             $parametros->recusaDeTramite = new stdClass();
             $parametros->recusaDeTramite->IDT = $idTramite;
-            $parametros->recusaDeTramite->justificativa = utf8_encode($justificativa);
+            $parametros->recusaDeTramite->justificativa = utf8_encode($objProcessoEletronicoRN->reduzirCampoTexto($justificativa, 1000));
             $parametros->recusaDeTramite->motivo = $motivo;
 
             $resultado = $this->tentarNovamenteSobErroHTTP(function($objPenWs) use ($parametros) {

